src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -583,9 +583,31 @@ export class CheckoutPageComponent extends Component {
       !isPaymentExpired
     );
 
-    const listingTitle = currentListing.attributes.title;
+    const listingTitle = currentListing ? currentListing.attributes.title : null;
```

Comment:
Listing prop is actually ensured already on line 520 - so, there's no need to use ternary.

Reply:
@OtterleyW btw. since you decided to move this loading-layout setup, maybe it's better to move it to somewhere near line 522